sdk/attestation/tests.yml

```diff
@@ -34,3 +33,5 @@ jobs:
        ATTESTATION_SERIALIZED_POLICY_SIGNING_KEY2: "$(ATTESTATION_SERIALIZED_POLICY_SIGNING_KEY2)"
        ATTESTATION_SERIALIZED_POLICY_SIGNING_KEY0: "$(ATTESTATION_SERIALIZED_POLICY_SIGNING_KEY0)"
        ATTESTATION_SERIALIZED_POLICY_SIGNING_KEY1: "$(ATTESTATION_SERIALIZED_POLICY_SIGNING_KEY1)"
+
```

Comment:
```suggestion
      AllocateResourceGroup: false
      BuildTargetingString: azure-security-attestation
      ServiceDirectory: attestation
      TestSamples: false
      DeployArmTemplate: true
```

Reply:
What does this do? I believe we need the tests to create the template in an RG, does this disable this?

Reply:
These are for when your tests run live in the pipelines, I know we're not worried about them now but better to be proactive. The last one will deploy the ARM template. I don't believe you need a resource group set-up outside of the one that will be deployed in the ARM template (disabled with `AllocateResourceGroup`), and for now won't test your samples live (we can enable later, but one thing at a time for now).